roseus/CMakeLists.txt

```diff
@@ -126,12 +126,10 @@ execute_process(COMMAND cmake -E copy ${PROJECT_SOURCE_DIR}/bin/roseus ${CATKIN_
 install(PROGRAMS bin/roseus
   DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION})
 # install
-install(PROGRAMS bin/roseus
-  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
 install(CODE "execute_process(COMMAND cmake -E make_directory \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CATKIN_GLOBAL_BIN_DESTINATION} RESULT_VARIABLE _mkdir_roseus_result OUTPUT_VARIABLE _mkdir_roseus_output)
               message(\"cmake -E make_directory \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CATKIN_GLOBAL_BIN_DESTINATION} returns \${_mkdir_roseus_result} ... \${_mkdir_roseus_output}.\")
-              execute_process(COMMAND cmake -E create_symlink ../${CATKIN_PACKAGE_BIN_DESTINATION}/roseus roseus WORKING_DIRECTORY \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CATKIN_GLOBAL_BIN_DESTINATION}/ RESULT_VARIABLE _install_roseus_result OUTPUT_VARIABLE _install_roseus_output)
-              message(\"create_symlink ../${CATKIN_PACKAGE_BIN_DESTINATION}/roseus roseus WORKING_DIRECTORY \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CATKIN_GLOBAL_BIN_DESTINATION}/ returns \${_install_roseus_result} ... \${_install_roseus_output}.\")")
+              execute_process(COMMAND cmake -E create_symlink ../${CATKIN_PACKAGE_SHARE_DESTINATION}/bin/roseus roseus WORKING_DIRECTORY \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CATKIN_GLOBAL_BIN_DESTINATION}/ RESULT_VARIABLE _install_roseus_result OUTPUT_VARIABLE _install_roseus_output)
+              message(\"create_symlink ../${CATKIN_PACKAGE_SHARE_DESTINATION}/bin/roseus roseus WORKING_DIRECTORY \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CATKIN_GLOBAL_BIN_DESTINATION}/ returns \${_install_roseus_result} ... \${_install_roseus_output}.\")")
```

Comment:
Sorry, I couldn't understand what you mean... What is this above diff ?
I think when we talked, we need to install roseus bin to global and set the link from share to global. Is that wrong?